### ISSUE TITLE
All red links don't work

### ISSUE THREAD

**Issue by SelenaCox** [freeCodeCamp/testable-projects-fcc#458]
#### Issue Description

I have almost completed the landing page project and it says I have 14/16, missing keeping the nav at the top and having at least one media query. 
When I click on the fails to see what has failed there is a lot of red text with both fails, and there are many links. All of these links lead me to 404 or 403. 
I also don't understand as my nav-bar is staying at the top. The three options are, the image and the header aren't.. I don't know if this is the problem. 

#### Browser Information

* Browser Name, Version: Google Chrome, Version: 67.0.3396.99 
* Operating System: macOS 10.13.5
* Mobile, Desktop, or Tablet: MacBook Air (13-inch, 2017)

#### Your Code / Link to Your Pen

 https://codepen.io/anon/pen/XBdmXv
 

HTML 
```  
<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
<div>
   <h id=header> Hi </h1>
<img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="original trombones logo">
  <div>
    <nav id="nav-bar">
    <ul>
      <li><a class="nav-link" href="#features">Features</a></li>
      <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
      <li><a class="nav-link" href="#pricing">Pricing</a></li>
        </ul>
  </nav>
  </div>
</div>
<section id="features">
</section>
<section id="how-it-works">
</section>
<section id="pricing">
    <div id="tenor">
      <div class="level">
        Tenor Trombone
      </div>
      <h2>$600</h2>
      <ol>
        <li>Lorem ipsum.</li>
        <li>Lorem ipsum.</li>
        <li>Lorem ipsum dolor.</li>
        <li>Lorem ipsum.</li>
      </ol>
      <button class="btn">Select</button>
    </div>
    <div id="bass">
      <div class="level">
        Bass Trombone
      </div>
      <h2>$900</h2>
      <ol>
        <li>Lorem ipsum.</li>
        <li>Lorem ipsum.</li>
        <li>Lorem ipsum dolor.</li>
        <li>Lorem ipsum.</li>
      </ol>
      <button class="btn">Select</button>
    </div>
    <div id="valve">
      <div class="level">
        Valve Trombone
      </div>
      <h2>$1200</h2>
      <ol>
        <li>Plays similar to a Trumpet</li>
        <li>Great for Jazz Bands</li>
        <li>Lorem ipsum dolor.</li>
        <li>Lorem ipsum.</li>
      </ol>
      <button class="btn">Select</button>
    </div>
  </section>

<iframe id="video" height="200" src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0" frameborder="0" allowfullscreen=""></iframe>

<form id="form" action="https://www.freecodecamp.com/email-submit">
  <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
  <input id=submit type=submit class=button> 
</form>
```

CSS
```  
#nav-bar {
    position: fixed;
    top: 0px;
    width: 100%;
  display: flex;
}​

#pricing {
display: flex;
}
```

**Comment by tbushman** [comment]
@SelenaCox Hello, so a couple of things that may confuse matters: There were three main errors in your code, before:  
```
<h id=header> Hi </h1>
<img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="original trombones logo">
<div>
    <nav id="nav-bar">
    <ul>
      <li><a class="nav-link" href="#features">Features</a></li>
      <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
      <li><a class="nav-link" href="#pricing">Pricing</a></li>
    </ul>
  </nav>  
```   
fixed:  
```  
<h1 id=header> Hi 
    <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="original trombones logo"/>
    <nav id="nav-bar">
        <ul>
          <li><a class="nav-link" href="#features">Features</a></li>
          <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
          <li><a class="nav-link" href="#pricing">Pricing</a></li>
        </ul>
    </nav>
</h1>  
```  
After you fix `<h id="header">` to be `<h1 id="header">`, you will see two new errors. The errors say that `img#header-img` and `nav#nav-bar` need to be child elements of `#header` as shown above. 
  
The last issue about the media query is a known issue being discussed in #437 so hopefully we'll figure something out for that but there's a work-around in the comments there.

**Comment by tbushman** [comment]
This should have been fixed with #534 . @SelenaCox Please open a new issue if you encounter further problems.
